### ISSUE TITLE
version 12.4.1 SvgCssUri component with width prop has a Type error

### ISSUE THREAD

**Issue by zhiqingchen** [react-native-svg/react-native-svg#1811]
# Question

SvgCssUri component has a Type error when i add width or height prop.
The following code is from [USAGE](https://github.com/react-native-svg/react-native-svg/blob/main/USAGE.md).

```jsx
import * as React from 'react';
import { SvgCssUri } from 'react-native-svg';

export default () => (
  <SvgCssUri
    width="100%"
    height="100%"
    uri="http://thenewcode.com/assets/svg/accessibility.svg"
  />
);
```

Type error

```
“Type '{ uri: string; width: string; height: string; }' is not assignable to type '“IntrinsicAttributes & { uri: string | null; } & AdditionalProps”'.ts(2322)
```

Type declare from source code

```ts
export declare function SvgCssUri(props: UriProps): JSX.Element;


export declare type UriProps = {
    uri: string | null;
} & AdditionalProps;

export declare type AdditionalProps = {
    onError?: (error: Error) => void;
    override?: Object;
};
```

**Comment by WoLewicki** [comment]
@zhiqingchen can you check if it is fixed in v12.4.2?

**Comment by zhiqingchen** [comment]
@WoLewicki  it fixed, but the `WithLocalSvg` component is still faulty, sorry for not submitting it together.

```js
<WithLocalSvg
  asset={src}
  width="100%"
  height="100%"
/>
```
